airflow/cli/commands/dag_command.py

```diff
@@ -351,8 +351,20 @@ def print_execution_interval(interval: DataInterval | None):
 @cli_utils.action_cli
 @suppress_logs_and_warning
 @providers_configuration_loaded
-def dag_list_dags(args) -> None:
+@provide_session
+def dag_list_dags(args, session=NEW_SESSION) -> None:
     """Display dags with or without stats at the command line."""
+    cols = args.columns if args.columns else []
+    not_valid_cols = [c for c in cols if c not in dag_schema.fields.keys()]
+    valid_cols = [c for c in cols if c in dag_schema.fields.keys()]
```

Comment:
```suggestion
    valid_cols = [c for c in cols if c in dag_schema.fields]
```

I think we might not need to call `keys()`, it seems to be the default behavior of a `dict`